src/Nest/Mapping/Types/Core/String/StringAttribute.cs

```diff
@@ -38,7 +41,7 @@ public class StringAttribute : ElasticsearchDocValuesPropertyAttributeBase, IStr
 		public bool Fielddata { get { return Self.FielddataUpgrade.GetValueOrDefault(true); } set { Self.FielddataUpgrade = value; } }
 		public bool EagerGlobalOrdinals { get { return Self.EagerGlobalOrdinals.GetValueOrDefault(true); } set { Self.EagerGlobalOrdinals = value; } }
 
-		[Obsolete("Only valid for indices created before Elasticsearch 5.0 and will be removed in the next major version.  For newly created indices, use Text or Keyword attribute instead.")]
+		[Obsolete("Only valid for indices created before Elasticsearch 5.0, removed in NEST 6.x.  For newly created indices, use Text or Keyword attribute instead.")]
```

Comment:
Maybe `removed in Elasticsearch 6.0`? It's clear that the feature is gone from Elasticsearch and the client reflects this?

Reply:
👍